src/stemmablenote.ts

```diff
@@ -253,7 +232,7 @@ export abstract class StemmableNote extends Note {
   }
 
   hasFlag(): boolean {
-    return Tables.getGlyphProps(this.duration).flag == true && !this.beam;
+    return this.glyphProps.codeFlagUp != undefined && !this.beam && !this.isRest();
```

Comment:
@rvilarl I'm going to go ahead and merge this, but in the next PR can you change this != to !== unless there is a great reason to use the looser inequality check?